docs/core/tools/dotnet-list-package.md

```diff
@@ -118,7 +118,11 @@ The project or solution file to operate on. If not specified, the command search
 
 - **`--vulnerable`**
 
-  Lists packages that have known vulnerabilities. Cannot be combined with `--deprecated` or `--outdated` options. Nuget.org is the source of information about vulnerabilities. For more information, see [Vulnerabilities](/nuget/api/registration-base-url-resource) and [How to Scan NuGet Packages for Security Vulnerabilities](https://devblogs.microsoft.com/nuget/how-to-scan-nuget-packages-for-security-vulnerabilities/).
+  Lists packages that have known vulnerabilities. Cannot be combined with `--deprecated` or `--outdated` options.
+  Use the `<AuditSources>` property in your configuration file to specify your source of vulnerabilities.
+  If `<AuditSources>` have not been specified, your specified `<PackageSources>` will be used for loading vulnerability data.
+  Note: The command uses [VulnerabilityInfo](/nuget/api/vulnerability-info) if you have `<AuditSources>` configured.
+  For more information, see [AuditSources](/nuget/concepts/auditing-packages#audit-sources) and [How to Scan NuGet Packages for Security Vulnerabilities](https://devblogs.microsoft.com/nuget/how-to-scan-nuget-packages-for-security-vulnerabilities/).
```

Comment:
Looks good. 
One important edit "source of vulnerabilities" -> "source of vulnerability data".
I think there's a simplification to combine the "Note:"...

```suggestion
  Use the `<AuditSources>` property in your configuration file to specify your source of vulnerability data which is acquired from the [VulnerabilityInfo](/nuget/api/vulnerability-info) resource.
  If `<AuditSources>` have not been specified, your specified `<PackageSources>` will be used for loading vulnerability data.
  For more information, see [AuditSources](/nuget/concepts/auditing-packages#audit-sources) and [How to Scan NuGet Packages for Security Vulnerabilities](https://devblogs.microsoft.com/nuget/how-to-scan-nuget-packages-for-security-vulnerabilities/).
```